.github/workflows/docs.yml

```diff
@@ -1,82 +1,64 @@
-name: Docs
+name: Deploy Website
 
-defaults:
-  run:
-    working-directory: ./docs
```

Comment:
Do we want to restore this for `website` and remove all the individual `working-directory: ./website` configs for each script command below?

Reply:
I like this idea.